### ISSUE TITLE
[User Story] Delivery of additional CNV and BAF files to Caesar

### ISSUE THREAD

**Issue by mathiasbio** [Clinical-Genomics/BALSAMIC#1493]
### Need

As a researcher / clinician doing interpretation of CNVs from tumor normal matched WGS analyses I would like to have access to:
- the BAF plots loaded into GENS but in a format accessible by IGV (because when working from home GENS can be too slow)
- the CNV.somatic.*.ascat.copynumber.txt.gz file but from the normal sample which can be useful to get the uniparental disomy

See freshdesk ticket with request: https://scilifelab.freshdesk.com/a/tickets/1000758

### Suggested approach

**For Ascat:** I don't know if Ascat produces this file from the normal sample, and I can't see if it is because all outputs from Ascat are saved in a temporary folder which is deleted on completion. 
- [ ] Verify that Ascat produces the file for the normal sample
- [ ] If above is true then we just need to save this and store + deliver it

**For BAF tracks in IGV:**: 
Maybe we can expand the functionality of the GENS script to have an IGV option where the BAF folder does not have the GENS specific zoom levels.  Then we can just add an additional rule, baf_to_igv or something, and use this script, then store an deliver. 

### System requirements assessed

- [ ] Yes, I have reviewed the system requirements

### Risk assessment needed

- [ ] Needed
- [ ] Not needed

**Comment by fevac** [comment]
possibly 20

**Comment by mathiasbio** [comment]
True! Thanks for pointing it out! I'll add what I did while testing
